src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,93 @@
+
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    public static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        user = new User();
+        user.setLogin("Evgenii");
+        user.setAge(34);
+        user.setPassword("1234567");
+    }
+
+    @AfterEach
+    public void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    public void register_nullLogin_NotOk() {
+        user.setLogin(null);
+        assertThrows(RuntimeException.class, () ->
+                registrationService.register(user));
+    }
+
+    @Test
+    public void register_NullPassword_NotOk() {
```

Comment:
First letter of each part of the method name must be in lower case
```suggestion
    public void register_nullPassword_notOk() {
```